Containerize process for local development setup (#8220)
opensearch-project/documentation-website: CONTRIBUTING.md

1. [Fork this repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo) and clone your fork.

> 2. Navigate to your cloned repository.

Reviewer: Please leave the 1. numbering. Jekyll automatically translates these into numbered lists. And it's easier to reorder/add/delete items this way.

Author: Reverted :D

1. Navigate to your cloned repository.

> ##### Building with locally-installed packages 

Reviewer: ```suggestion
##### Building using locally-installed packages 
```

   bundle install
   ```

> ##### Building with containerization

Reviewer: ```suggestion
##### Building using containerization
```

> 1. Assuming you have `docker-compose` installed, execute

Reviewer: ```suggestion
Assuming you have `docker-compose` installed, run the following command:
```
